tests/trestlebot/cli/test_sync_cac_content_cmd.py

```diff
@@ -148,6 +148,14 @@ def test_sync_product(tmp_repo: Tuple[str, Repo]) -> None:
     ]
     assert set_params_dict["var_sshd_set_keepalive"] == ["1"]
     assert set_params_dict["var_system_crypto_policy"] == ["fips"]
+    # Test the control status is populated to implemented_requirements
```

Comment:
Nit: I might suggest adding a few more test cases to check the `OSCALStatus` class is mapping the values as expected. The string value for `not-applicable` are the same in SSG and OSCAL I believe.

Reply:
Thanks. That's a good suggestion. I updated the test cases.